src/_P109_ThermOLED.ino

```diff
@@ -240,16 +240,16 @@ boolean Plugin_109(uint8_t function, struct EventStruct *event, String& string)
       P109_CONFIG_HYSTERESIS     = (getFormItemInt(F("hyst")) / 10.0f);
       P109_CONFIG_TIMEOUT        = getFormItemInt(F("timeout"));
       P109_CONFIG_MAX_TIMEOUT    = getFormItemInt(F("timeout_max"));
-      P109_CONFIG_STEP_TIMEOUT   = getFormItemInt(F("timeout_step"));
       P109_CONFIG_SETPOINT_DELAY = getFormItemInt(F("setpdelay")) + P109_SETPOINT_OFFSET;
+      P109_CONFIG_STEP_TIMEOUT   = getFormItemInt(F("timeout_step"));
       P109_MODE_STATE_INITIAL    = getFormItemInt(F("mode"));
       uint32_t lSettings = 0u;
-      bitWrite(lSettings, P109_FLAG_TASKNAME_IN_TITLE, isFormItemChecked(F("ptask")));
-      bitWrite(lSettings, P109_FLAG_ALTERNATE_HEADER,  !isFormItemChecked(F("palt"))); // Inverted
-      bitWrite(lSettings, P109_FLAG_RELAY_INVERT,      isFormItemChecked(F("invertrelay")));
-      bitWrite(lSettings, P109_FLAG_BUTTON1_NO_INTPULLUP,      isFormItemChecked(F("NoIntPullupButton1")));
-      bitWrite(lSettings, P109_FLAG_BUTTON2_NO_INTPULLUP,      isFormItemChecked(F("NoIntPullupButton2")));
-      bitWrite(lSettings, P109_FLAG_BUTTON3_NO_INTPULLUP,      isFormItemChecked(F("NoIntPullupButton3")));
+      bitWrite(lSettings, P109_FLAG_TASKNAME_IN_TITLE,   isFormItemChecked(F("ptask")));
+      bitWrite(lSettings, P109_FLAG_ALTERNATE_HEADER,    !isFormItemChecked(F("palt"))); // Inverted
+      bitWrite(lSettings, P109_FLAG_RELAY_INVERT,        isFormItemChecked(F("invertrelay")));
+      bitWrite(lSettings, P109_FLAG_BUTTON1_INT_PULL_UP, !isFormItemChecked(F("IntPullupButton1"))); // Inverted
```

Comment:
This "inverted" stuff is exactly what I was afraid of that would happen.
This is extremely error prone and a nightmare to debug or verify.